crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -54,19 +85,100 @@ pub struct TonemappingPipeline {
     texture_bind_group: BindGroupLayout,
 }
 
-#[derive(Copy, Clone, PartialEq, Eq, Hash)]
+/// Optionally enables a tonemapping shader that maps the high dynamic range (HDR) image data into a low dynamic range (LDR) image for a given [`Camera`] entity.
```

Comment:
Theoretically, in the future when we support HDR output, it would be HDR -> a smaller subset of HDR suitable to the display. Not just HDR -> LDR.

Reply:
From the perspective of a display transform a "HDR" display is still a LDR device. Unfortunately LDR/HDR are used to mean a bunch different mostly unrelated things. I'll try to find better phrasing to avoid this confusion.